src/js/letters/actions/letters.js

```diff
@@ -2,28 +2,28 @@ import Raven from 'raven-js';
 
 import { apiRequest } from '../utils/helpers.jsx';
 import {
+  ADDRESS_TYPES,
   BACKEND_AUTHENTICATION_ERROR,
   BACKEND_SERVICE_ERROR,
-  GET_LETTERS_FAILURE,
-  GET_LETTERS_SUCCESS,
+  GET_ADDRESS_COUNTRIES_SUCCESS,
+  GET_ADDRESS_COUNTRIES_FAILURE,
   GET_ADDRESS_FAILURE,
   GET_ADDRESS_SUCCESS,
+  GET_ADDRESS_STATES_FAILURE,
+  GET_ADDRESS_STATES_SUCCESS,
   GET_BENEFIT_SUMMARY_OPTIONS_FAILURE,
   GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS,
+  GET_LETTERS_FAILURE,
```

Comment:
Again, as long as we're re-ordering these, we can move this to right above `GET_LETTERS_SUCCESS`.

Reply:
yep!